packages/wallets/src/routes/WalletsListingRoute/WalletsListingRoute.scss

```
@@ -5,10 +5,10 @@
     flex-direction: column;
     align-items: center;
     justify-content: center;
-    gap: 2.4rem;
     background-color: var(--general-main-1, #ffffff);
 
     @include desktop {
         padding: 4rem;
+        padding-top: 0;
```

```suggestion
        padding: 0 4rem;
```
Probably won't need the top and bottom padding with 2.4rem gap as well.

---

Updated styling